### DISCUSSION TITLE
Crazyflie loco positioning system not showing lps anchor co-ordinates

### DISCUSSION THREAD

**Discussion by AT882** [bitcraze/discussions#1674] [Q&A]
I have positioned the anchors and powered them ON, the issue is that its not showing co-ordinates in the cfclient the anchors are detected as green indications are there but no coordinates
i am using 6 anchor arrangement. and upgraded the firmware and done the numbering of anchors as well in TWR mode
a screenshot is attached for better understanding.
any help is appreciated.
![Screenshot 2024-11-27 153022](https://github.com/user-attachments/assets/a462497e-10ca-4740-8fff-703ac048f16e)

**Comment by Bananazooka** [comment]
Hi! Could you show us what it looks like when you switch the "graph settings" mode back to "position estimate"? When you move the Crazyflie around, does the graph show the movement?
You're also missing one of your anchors - maybe you should retry configuring the system.
